src/app/pages/forms-and-validation/verify-on-submit/verify-on-submit.component.html

```diff
@@ -0,0 +1,66 @@
+<mat-toolbar color="primary" class="app-bar mat-elevation-z2" data-cy="pxb-toolbar">
+    <button data-cy="toolbar-menu" *ngIf="isSmall" style="margin-left: -8px" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Fixed Length Passcode</div>
+</mat-toolbar>
+
+<pxb-slide-panel [activePane]="isLeftPaneVisible ? 'left' : 'right'">
+    <div leftPane>
+        <section class="find-device" [class.small]="isSmall">
```

Comment:
Is this small class used anywhere? I couldn't find it in the local stylesheet or global styles.  If not, the `isSmall` var can be removed from the component as well.

Reply:
Removed it, since not using this class anywhere in current component.